AmericanMedicalAssociation/joe-style-guide
# EWL-9806: Update gulp-sass compiler and update deprecated code.

## Ticket(s)

Please do not submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**

- [Issue XXX: Issue Title](https://github.com/AmericanMedicalAssociation/joe-style-guide/issues/XXX)

**Jira Ticket**

- [EWL-9806: Update scss for new gulp-sass compiler](https://issues.ama-assn.org/browse/EWL-9806)


## Description

Update sass gulp compiler and dependencies.


## To Test

- within styleguide run lando npm install
- confirm running lando gulp serve works
- link styleeguide locally and confirm styles still compile and update locally

## Visual Regressions

Does your work introduce any visual regressions to existing work in the Style Guide, or in a Drupal 8 environment? Please either call these out here or address them before marking your PR as `ready for review`.


## Relevant Screenshots/GIFs

Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks

Remaining tasks?


## Additional Notes

Anything more to add? Good things to call out here are:
- are there any PRs in this or other repositories that relate to or affect this PR?
- are there any caveats or oddities testers should know about when testing this PR?
- are there any problems you ran into during your work that you'd like to call out?
